### PR TITLE
Tally Reports Part 2 - UI components

### PR DESCRIPTION
**Pull request by adghayes**
__review commit-by-commit, read description below, and feel free to play around with component in storybook__

## Overview

Part of #3452. 

**Commit 1** - File reorganization

**Commit 2** - Update snapshots

**Commit 3** - Creates a `ContestResultsTable` component to eventually fully replace the `ContestTally` component. The component differs in that it relies on the new types, it removes an "dimming" functionality for irrelevant contests. It represents only one contest, and it parent components are responsible for listing and choosing the contests, unlike in `ContestTally`. Styles are just copied, and there was no iteration on styling.

**Commit 4** - Creates an `AdminTallyReport` component. I've always been frustrated that our tally report existed not as a UI component but as one of several return results from the `ElectionManagerTallyReport` component in VxAdmin. I'm excited to use `storybook` to make viewing and iterating on these reports easier. Component mostly copied from `ElectionManagerTallyReport`, styling mostly untouched

**Commit 5** - Add card counts component to show card counts on reports instead of voting method breakdown.

## Demo Video or Screenshot

Screen Shot 2023-06-20 at 9 44 37 AM
Screen Shot 2023-06-20 at 9 44 30 AM

### CODE REVIEW

**Comment by adghayes** [comment]
@kofi-q I realized I still need to write some testing on this 😅 so give this one a second.

**Comment by adghayes** [comment]
@kofi-q ready to go 👍

File: libs/ui/jest.config.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts'],
   coveragePathIgnorePatterns: [
+    'src/reports/index.ts',
```
Comment (kofi-q) [inline libs/ui/jest.config.js:11]: weird - is coverage complaining about an index file with only `export` statements?

**Review by kofi-q — APPROVED** [review]
🎉

File: libs/ui/jest.config.js (hunk 1 above)
Comment (adghayes) [reply libs/ui/jest.config.js:11]: Yeah :(

**Pull request merged**